src/main/java/pb360/RestApiApplication.java

```diff
@@ -2,8 +2,10 @@
 
 import org.springframework.boot.SpringApplication;
 import org.springframework.boot.autoconfigure.SpringBootApplication;
+import org.springframework.boot.autoconfigure.web.servlet.error.ErrorMvcAutoConfiguration;
 
-@SpringBootApplication
+
+@SpringBootApplication(exclude = {ErrorMvcAutoConfiguration.class})
```

Comment:
please explain about exclude = {ErrorMvcAutoConfiguration} on the status mail today